### PR TITLE
Upgrade packages and dependencies across all workspace packages

### PR DESCRIPTION
**Pull request by asimpk**
This PR is review and sync our approach to upgrade workspace packages dependencies.

### CODE REVIEW

File: docs/sdk/lib-base.asset_tokens.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 Assets Types
 
-<b>Signature:</b>
+**Signature:**
```
Comment (marceljay) [inline docs/sdk/lib-base.asset_tokens.md:9]: If this happened because of a single package changing the API/SDK documentation rules, i'd prefer this upgrade to be a separate commit.

Comment (marceljay) [inline docs/sdk/lib-base.asset_tokens.md:9]: If these changes occur due to a single package being upgraded (it must have run too?) than i prefer this to be it's own PR, because it clutters this one here.

**Review by marceljay — COMMENTED** [review]
Please in a first step rebase and remove merge commits.

**Comment by marceljay** [comment]
@asimpk I think it's important to make incremental progress. If you managed to fix everything without first migrating to yarn v3, we can end the PR there, and have yarn v3 migration in a separate PR.

File: docs/sdk/lib-base.asset_tokens.md (hunk 1 above)
Comment (asimpk) [reply docs/sdk/lib-base.asset_tokens.md:9]: Yeah these are many files and  should be in seperate PR.

**Comment by asimpk** [comment]
> @asimpk I think it's important to make incremental progress. If you managed to fix everything without first migrating to yarn v3, we can end the PR there, and have yarn v3 migration in a separate PR.

My intent was to sync my approach with team. I'm closing this PR and will finish this upgrading dependencies task in 2, 3 PR's. 
Here is the first PR https://github.com/kumodao/kumo-protocol/pull/405
Next will be address these:
Upgrade yarn version to V3
Integrate back skipped packages (Subgraph, lib-subgraph, fuzzer)
Review the typescript changes, general to specific implementation.

File: docs/sdk/lib-base.asset_tokens.md (hunk 1 above)
Comment (marceljay) [reply docs/sdk/lib-base.asset_tokens.md:9]: @asimpk
